Python/bytecodes.c

```diff
@@ -3930,30 +4377,34 @@ dummy_func(
                     frame->return_offset = 1;
                     DISPATCH_INLINED(new_frame);
                 }
-                result = PyObject_Call(func, callargs, kwargs);
+                result = PyStackRef_FromPyObjectSteal(PyObject_Call(func, callargs, kwargs));
             }
             DECREF_INPUTS();
-            assert(PEEK(2 + (oparg & 1)) == NULL);
-            ERROR_IF(result == NULL, error);
+            assert(PyStackRef_AsPyObjectBorrow(PEEK(2 + (oparg & 1))) == NULL);
+            ERROR_IF(PyStackRef_IsNull(result), error);
             CHECK_EVAL_BREAKER();
         }
 
-        inst(MAKE_FUNCTION, (codeobj -- func)) {
+        inst(MAKE_FUNCTION, (codeobj_st -- func)) {
+            PyObject *codeobj = PyStackRef_AsPyObjectBorrow(codeobj_st);
```

Comment:
```suggestion
            PyObject *codeobj = PyStackRef_AsPyObjectSteal(codeobj_st);
```

Reply:
This doesn't seem right. `PyFunction_New` takes a borrowed ref.